types/rc-switch/rc-switch-tests.ts

```diff
@@ -3,17 +3,12 @@ import * as ReactDOM from 'react-dom';
 import Switch, { Props } from 'rc-switch';
 
 /**
- * Only required props.
+ * Only required props (none)
```

Comment:
For anyone reading it in the future: `SwitchProps` defined in [the source code](https://github.com/react-component/switch/blob/32bd2706dc3ceee0c2fbdbd7ea6f6dbe79354636/src/Switch.tsx#L9-L23) require two props: `onMouseUp` and `loadingIcon`. In this PR, however, no property is required.

This reflects what is actually happening in the source code. They both could as well be made optional because:

1. `onMouseUp` [is called only when it exists](https://github.com/react-component/switch/blob/32bd2706dc3ceee0c2fbdbd7ea6f6dbe79354636/src/Switch.tsx#L108-L110)
1. `loadingIcon` [is allowed be `undefined`](https://github.com/react-component/switch/blob/32bd2706dc3ceee0c2fbdbd7ea6f6dbe79354636/src/Switch.tsx#L156)

This puts type definitions in alignment with documentation which claims you can use `<Switch />` without passing any props (`ReactDOM.render(<Switch />, container)`).